index.html

```diff
@@ -0,0 +1,219 @@
+        </p>
+        <p><br><a class="intro-connect">LET’S CONNECT</a><br></p>
+        <ul class="view-social-midea">
+            <li class="social-icon">
+                <a><img src="./assets/images/social-icon/twitter-icon.png" alt="twitter"></a>
+            </li>
+            <li class="social-icon">
+                <a><img src="./assets/images/social-icon/linkedin icon.png" alt="linkedin"></a>
+            </li>
+            <li class="social-icon">
+                <a><img src="./assets/images/social-icon/m-icon.png" alt="m"></a>
+            </li>
+            <li class="social-icon">
+                <a><img src="./assets/images/social-icon/github-icon.png" alt="github"></a>
+            </li>
+            <li class="social-icon">
+                <a><img src="./assets/images/social-icon/clip-icon.png" alt="fingers"></a>
+            </li>
+        </ul>
+            </span>
+        </section>
+
+         <section class="grid-item grid-item-3 work-section">
+            <div id="work-container">
+                <span class="work-content">
+                <div class="img-content">
+                  <img src="./assets/images/work-section/project1.png" alt="work 1" class="" />
+                </div>
+                  <h1 class="work-title">Tonic</h1>
+                  <div class="work-role">
+                    <h3 class="work-role-list role-job ">CANOPY</h3>
+                    <div class="work-role-list  role-dot">&#8901;</div>
+                    <div class="work-role-list  role-name">Back end dev</div>
+                    <div class="work-role-list  role-dot">&#8901;</div>
+                    <div class="work-role-list  role-year">2023</div>
+                  </div>
+                  <p class="project-description">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                    <ul class="my-skill">
+                        <li><button title="html">html</button></li>
+                        <li><button title="css">css</button></li>
+                        <li><button title="js">Javascript</button></li>
+                    </ul>
+                    <span class="p-more-information">
+                      <a class="moresee-link">See project</a>
+                      </span>
+                </span>
+                <!-- -------------------1 -->
+                <span class="work-content">
+                    <div class="img-content">
+                      <img src="./assets/images/work-section/project2.png" alt="work 1" class="" />
+                    </div>
+                      <h1 class="work-title">Multi-Post<br>
+                        Stories</h1>
+                      <div class="work-role">
+                        <h3 class="work-role-list role-job ">CANOPY</h3>
+                        <div class="work-role-list  role-dot">&#8901;</div>
```

Comment:
- [ ] KIndly remove all the visible black dots from all the project cards to match the figma design. YOu can actually create the dot effect by styling an empty div as you have done.

your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/219848646-f492d1bd-7099-4bde-bc2f-7e82a805e193.png)
 
figma design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/219848682-0b18a61d-2d30-4173-8ae2-41f8fef2acc2.png)